Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -192,9 +192,9 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
                 make.leading.equalTo(self.lockImageView.snp_trailing).offset(locationContentInset)
             }
             if readerModeButton.hidden {
-                make.trailing.equalTo(container.snp_trailing)
+                make.trailing.equalTo(container.snp_trailing).offset(-locationContentInset)
```

Comment:
Do we need a whole 8px space here, or should we use `-(locationContentInset / 2)` like we do with the `readerModeButton`?